docs/src/pages/[platform]/getting-started/accessibility/intro.mdx

```diff
@@ -0,0 +1,12 @@
+import { Alert } from '@aws-amplify/ui-react';
+
+Amplify UI components follow [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/) and 
```

Comment:
Can we make these links open up in a new tab? Perhaps we can use our Link primitive and the `isExternal` prop

Reply:
Yeah, that makes sense. Updated in latest.